### PR TITLE
PCC01 Artemsav

### PR DESCRIPTION
**Pull request by Artemsav**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): [5]
Estimated time spent (hours): [3]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): [The most difficult problem for me surprisingly was git. I don't quite sure understand how it works. Anyway it was fun]
Other feedback (what can we improve?): [You gave whatever you could give, thank you]

### CODE REVIEW

**Comment by bbelderbos** [comment]
The PR looks good, any issues with our new instructions / video?

**Comment by Artemsav** [comment]
Nope, everything all right if you do it in linux-system. But at first I worked in windows(by the way I used pycharm) and I am not quite understand how should I make a branch. Anyway, now I've understood, that's some book about git would be useful for me.

**Comment by Artemsav** [comment]
Thank you!

File: 01/Artemsav/wordvalue.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+def load_words():
+    with open('dictionary.txt', 'r') as dictionary:
+        data = dictionary.read()
+        elsp = data.split()
+    return elsp
+
+def calc_word_value(word):
+    finish_count = 0
+    for k in list(word):
```
Comment (pybites) [inline 01/Artemsav/wordvalue.py]: I would use `word` instead of `k`

File: 01/Artemsav/wordvalue.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    with open('dictionary.txt', 'r') as dictionary:
+        data = dictionary.read()
+        elsp = data.split()
+    return elsp
```
Comment (pybites) [inline 01/Artemsav/wordvalue.py]: `elsp`? maybe `words`?

**Comment by pybites** [comment]
Good work

**Pull request merged**